src/app/client/src/app/modules/player-helper/components/player/player.component.html

```diff
@@ -29,12 +29,14 @@
         </svg>
     </div>
   </div>
-    <!-- <div > -->
-      <div id="pdfPlayer" class="contentViewerIframeShadow" [ngClass]="{'player-fullscreen content-full-video': isFullScreenView}">
-        <sunbird-pdf-player 
-        [playerConfig]="playerConfig" (playerEvent)="pdfEventHandler($event)"
-        (telemetryEvent)="generateContentReadEvent($event, true)"
-        ></sunbird-pdf-player>
+      <div class="contentViewerIframeShadow" [ngClass]="{'player-fullscreen content-full-video': isFullScreenView}">
+        <sunbird-pdf-player *ngIf="playerConfig.metadata.mimeType === 'application/pdf'" [playerConfig]="playerConfig"
+          (playerEvent)="eventHandler($event)" (telemetryEvent)="generateContentReadEvent($event, true)">
+        </sunbird-pdf-player>
+
+        <sunbird-video-player *ngIf="playerConfig.metadata.mimeType === 'video/mp4'" [playerConfig]="playerConfig"
```

Comment:
mimeType may not be only mp4 it can be others also check with KP team once
we should use all video types other than youtube

Reply:
Done